### DISCUSSION TITLE
Invalid Handle and bsky.network 503

### DISCUSSION THREAD

**Discussion by sapph42** [bluesky-social/atproto#3077] [Dev Debugging]
After successfully setting up a PDS, and successfully registering on bsky.app with an invite code from my PDS, I noticed irregularities.  My profile says Invalid Handle, and editing my handle does nothing.  Replies don't show up (but are visible in PDSls).  I was advised to run pdsadmin request-crawl on my PDS, which returns 

Requesting crawl from https://bsky.network
curl: (22) The requested URL returned error: 503

I presume the 503 from crawl is related to the issues I'm having using BlueSky through the web interface.  Is there a known outage that is causing this to occur? Is the 503 related to some misconfiguration on my PDS? Would a 503 from bsky.network reasonably cause the issues I'm seeing through the web interface?

**Comment by zombxx** [comment]
I have a similar issue. Did you ever get a fix?

**Reply by sapph42** [reply]
The 503 issue seemed to be an outage.
